### ISSUE TITLE
[Fluff] no (fluff) mech images shown in HQ

### ISSUE THREAD

**Issue by kernhe** [MegaMek/mekhq#3394]
### Environment
V: 0.49.9 Snapshot
O: linux (Ubuntu 20.04.4)
J: openjdk 11.0.15 2022-04-19

### Description
I have plenty of (fluff) images for various mechs, but they are no longer displayed in HQ.
Is the file name now case sensitive (known linux issue)?

**Comment by HammerGS** [comment]
There is a new folder structure for fluff images.

![image](https://user-images.githubusercontent.com/13752452/179017734-df303f7b-0871-4e33-bb14-61f529984bb2.png)

If you want to test with this structure.

**Comment by kernhe** [comment]
my structure is similar (different sorting due to case sensitivity):
`├── aerospace
├── armor
├── Art
│   ├── Aero
│   ├── BattleArmor
│   ├── mech
│   ├── protomech
│   └── Vehicle
├── BattleArmor
├── dropship
├── jumpship
├── mech
├── mech_attic
├── proto
├── protomech
├── station
├── vehicle
└── warship
`

mech_attic is just a folder, where I store the older images, just for nostalgia

**Comment by HammerGS** [comment]
Right, but the new code looks up the structure I posted. Is it all images gone missing or certain unit types?

**Comment by kernhe** [comment]
Currently no image is shown in HQ.

In the folder "mech" I have 460 files, ... yes, it is a bit a mess, because some file are available as jpg, png and gif, some only as png.  I was gathering all images I was able to get hands on.  e.g. I have Javlin.png and Javlin.gif (both different and in the past I think the gif was shown)

**Comment by HammerGS** [comment]
Just tested in MekHQ from the Repo. Might be something specific to Linux. 

![image](https://user-images.githubusercontent.com/13752452/179031928-4ffed8cc-bac5-4c67-93b6-449c688f50b1.png)

**Comment by HammerGS** [comment]
Someone did some testing and looks like its a case sensitive issue.

**Comment by Windchild292** [comment]
Linux is a case sensitive file system, so an incorrect case will cause issues. Our internal file is misnamed, so I'll try to push up a fix. Transferring to MM as that's the proper tracker.
